x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -232,4 +234,35 @@ export function resultsServiceRoutes({ router, mlLicense }: RouteInitialization)
       }
     })
   );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {post} /api/ml/results/anomaly_search Performs a search on the anomaly results index
+   * @apiName AnomalySearch
```

Comment:
You should also add `AnomalySearch` to the `ResultsService` group in the apidoc.json file to ensure it gets added to the API docs.

Reply:
Updated in 5483dd0